tests/block/create_block_test.go

```diff
@@ -21,35 +21,58 @@ import (
 	"testing"
 
 	"github.com/rook/rook/pkg/model"
-	"github.com/rook/rook/tests"
 	"github.com/rook/rook/tests/framework/clients"
 	"github.com/rook/rook/tests/framework/contracts"
+	"github.com/rook/rook/tests/framework/enums"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/objects"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
+
+	"fmt"
+	"os"
 )
 
 var (
-	defaultPool   = "rbd"
+	defaultPool    = "rbd"
 	pool1          = "rook_test_pool"
 	blockImageName = "testImage"
+	Env            objects.EnvironmentManifest
+	rook           *installer.InstallHelper
 )
 
 func TestBlockCreate(t *testing.T) {
 	suite.Run(t, new(BlockImageCreateSuite))
 }
 
+
 type BlockImageCreateSuite struct {
 	suite.Suite
-	testClient      *clients.TestClient
-	rc              contracts.RestAPIOperator
+	testClient     *clients.TestClient
+	rc             contracts.RestAPIOperator
 	initBlockCount int
 }
 
 func (s *BlockImageCreateSuite) SetupSuite() {
 
 	var err error
+	Env = objects.NewManifest()
+	// this is going to be cleaned up after #774
+	if kv := os.Getenv("KUBE_VERSION"); kv != "" {
```

Comment:
Could you have two helper methods, one for install and one for install? Then each suite would just have a 1-liner for install and 1-liner for uninstall. That would remove a lot of duplication with checking env vars and such.